src/api/exposure.js

```diff
@@ -1,20 +1,33 @@
 //const debug = require("debug")("mgmt");
 
-const EXPOSURE_API_ENDPOINT = "https://exposureapi.emp.ebsd.ericsson.net/v2";
-
+const EXPOSURE_API_ENDPOINT_PROD =
+  "https://exposureapi.emp.ebsd.ericsson.net/v2";
+const EXPOSURE_API_ENDPOINT_STAGE =
+  "https://psempexposureapi.ebsd.ericsson.net/v2";
```

Comment:
I don't know if we should have `v2` as a part of these urls even though all endpoint as of right now might have it 🤔

Reply:
Agree. I did not want to change it myself though, since I don't really understand why it is that way.

Reply:
Ah, I'll look into it then